File: NetFramework/Schedule/VB/CustomForms/CopyForm.vb
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Text
Imports System.Windows.Forms

Namespace TestSchedule
	''' <summary>
	''' This form allows to select DateTime from the MonthCalendar control.
	''' </summary>
	Public Partial Class CopyForm
		Inherits Form
		Private _date As DateTime = DateTime.Today

		Public Sub New()
			InitializeComponent()
		End Sub

		Public ReadOnly Property [Date]() As DateTime
			Get
				Return _date
			End Get
		End Property

		Private Sub monthCalendar1_DateSelected(ByVal sender As Object, ByVal e As DateRangeEventArgs) Handles monthCalendar1.DateSelected
			_date = e.Start
			Me.DialogResult = DialogResult.OK
			Close()
		End Sub
	End Class
End Namespace

File: NetFramework/Schedule/CS/CustomData/CopyForm.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;

namespace CustomData
{
	/// <summary>
	/// This form allows to select DateTime from the MonthCalendar control.
	/// </summary>
	public partial class CopyForm : Form
	{
		DateTime _date = DateTime.Today;

		public CopyForm()
		{
			InitializeComponent();
		}

		public DateTime Date
		{
			get
			{
				return _date;
			}
		}

		private void monthCalendar1_DateSelected(object sender, DateRangeEventArgs e)
		{
			_date = e.Start;
			this.DialogResult = DialogResult.OK;
			Close();
		}
	}
}